kolibri/content/serializers.py

```diff
@@ -48,9 +49,15 @@ def get_ancestors(self, target_node):
         """
         return target_node.get_ancestors().values('pk', 'title')
 
+    def get_thumbnail(self, target_node):
+        try:
+            return target_node.files.get(thumbnail=True).get_url()
```

Comment:
When I woke up this morning, I realized you would make this comment. Will add after call.